src/Symfony/Component/Translation/Translator.php

```diff
@@ -132,6 +132,10 @@ public function addResource($format, $resource, $locale, $domain = null)
             $domain = 'messages';
         }
 
+        if (null === $locale) {
+            @trigger_error(sprintf('Passing "null" to "$locale" in %s has been deprecated since Symfony 4.4 and will throw in 5.0.', __METHOD__), E_USER_DEPRECATED);
```

Comment:
```suggestion
            @trigger_error(sprintf('Passing "null" to "$locale" in %s has been deprecated since Symfony 4.4 and will throw an exception in 5.0.', __METHOD__), E_USER_DEPRECATED);
```